src/Attribute/Parameter/DiResolver.php

```diff
@@ -17,12 +24,55 @@ public function __construct(
     ) {
     }
 
+    /**
+     * @throws ContainerExceptionInterface
+     * @throws DiNotFoundException
+     * @throws NotResolvedException
+     */
     public function getParameterValue(ParameterAttributeInterface $attribute, Context $context): mixed
     {
         if (!$attribute instanceof Di) {
             throw new UnexpectedAttributeException(Di::class, $attribute);
         }
 
-        return $this->container->get($attribute->getId());
+        $id = $attribute->getId();
+        if ($id !== null) {
+            return $this->container->get($id);
```

Comment:
Shouldn't it be wrapped with try/catch like on line 48?
There `NotFoundExceptionInterface` is converted to `NotResolvedException`/`DiNotFoundException`

Reply:
Yes

Reply:
Done